File: tests/openmp_examples/program_control/reproducible.2.f90
!$omp    parallel
!$omp       do order(reproducible: concurrent)
!$omp       end do nowait
!$omp       do order(reproducible: concurrent)
!$omp    end parallel
!$omp    parallel
!$omp       do schedule(static) order(concurrent)
!$omp       end do nowait
!$omp       do schedule(static) order(concurrent)
!$omp    end parallel
!$omp    parallel
!$omp       do schedule(static) order(unconstrained: concurrent)
!$omp       do schedule(static) order(unconstrained: concurrent)
!$omp    end parallel

File: tests/openmp_examples/program_control/reproducible.2.c
#pragma omp parallel
#pragma omp for order(reproducible: concurrent) nowait
#pragma omp for order(reproducible: concurrent)
#pragma omp parallel
#pragma omp for schedule(static) order(concurrent) nowait
#pragma omp for schedule(static) order(concurrent)
#pragma omp parallel
#pragma omp for schedule(static) order(unconstrained: concurrent)
#pragma omp for schedule(static) order(unconstrained: concurrent)
